index.html

```diff
@@ -2009,6 +2016,9 @@ <h3>Processing capabilities</h3>
 
    <dt>"<code>userAgent</code>"
    <dd>String containing the <a>default User-Agent value</a>.
+
+   <dt>"<code>processID</code>"
```

Comment:
For example, this could be:

```suggestion
   <dt>"<code>processID</code> (optional)"
```

Reply:
Right; do we want to say anything here about suggesting that the processID be included iff the client is on localhost?